srv/components/misc_pkgs/kafka/config.sls

```diff
@@ -39,6 +39,26 @@ Update zoopkeeper cofig:
     - template: jinja
     - backup: '.bak'
 
+Update permissions for datadir:
+  file.directory:
+    - name: /var/lib/zookeeper
+    - user: kafka
+    - group: kafka
+    - makedirs: True
+    - recurse:
+      - user
+      - group
+
+Update permissions for datalogdir:
```

Comment:
Set right permissions if we specify users.

Preferred:
755/644

Reply:
have set 755 as dir_mode